app/services/create_work_package_service.rb

```diff
@@ -43,7 +43,7 @@ def create
       author: user,
       type: project.types.where(is_default: true).first || project.types.first
     }
-    self.work_package = project.add_work_package(hash)
```

Comment:
good catch! I thought, that we stopped remembering the own work package, long ago...